modules/core/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -127,6 +129,8 @@
 
     public static final String INFO_USE_S3_ACCELERATE = "useS3Accelerate";
 
+    public static final Pattern CHARSET_CONTENT_TYPE_PATTERN = Pattern.compile("(.*);.*");
```

Comment:
In theory this is not generic enough, for instance this is a valid `Content-Type` header: `text/plain  ;  CHARSET = UTF-8 ; foo=bar`